### PR TITLE
Weekly merge

### PR DESCRIPTION
**Pull request by toniocodo**
This PR brings:
- self hosted fonts for better TTI performance
- first features for unified dApp
- latest graphql schema updates

### CODE REVIEW

File: libs/defi/oeth/src/swap/actions/swapZapperEth.ts
Comment (apexearth) [inline libs/defi/oeth/src/swap/actions/swapZapperEth.ts:1]: is this just a copy from another place? seems duplicative?

File: libs/defi/ousd/src/swap/actions/flipper.ts
Comment (apexearth) [inline libs/defi/ousd/src/swap/actions/flipper.ts:1]: if this stuff exists in libs/ousd and also libs/defi/ousd then I'd say it doesn't belong in either 🙈

**Review by apexearth — APPROVED** [review]
ok, commented on some concerns

File: libs/defi/oeth/src/swap/actions/swapZapperEth.ts
Comment (toniocodo) [reply libs/defi/oeth/src/swap/actions/swapZapperEth.ts:1]: yes, this is totally a copy from another place! There's no point in sharing those and eventually, only the routes from the unified dApp will be used

File: libs/defi/ousd/src/swap/actions/flipper.ts
Comment (toniocodo) [reply libs/defi/ousd/src/swap/actions/flipper.ts:1]: eventually we'll deprecate ousd and oeth dApp, so the routes will go with them. In the meantime, it is totally fine to have those duplicates

**Pull request merged**